### DISCUSSION TITLE
Is there recommended version or version compatibility with EKS Cluster?

### DISCUSSION THREAD

**Discussion by pkalsh** [aws/amazon-vpc-cni-k8s#2261] [Q&A]
Hi, 
Is there recommended version or version compatibility with EKS Cluster? 
I want to upgrade my EKS cluster version up to 1.24

thanks

**Comment by vchintal** [comment]
Not sure if this answers your question: https://docs.aws.amazon.com/eks/latest/userguide/managing-vpc-cni.html

**Comment by joebowbeer** [comment]
I've asked for clarification regarding any difference between the recommended version and the default version:

https://github.com/aws/amazon-vpc-cni-k8s/issues/2747
